Formula/wla-dx.rb

```diff
@@ -14,13 +14,22 @@ class WlaDx < Formula
     sha256 "a78744385e31896de6b34ae3426c6081ada2f28b997f0c5c15555e10bd03d53e" => :mountain_lion
   end
 
+  if build.head?
```

Comment:
I wasn't aware that was possible; I tried to find if it was but I guess I missed it. This is what I mean when I say I don't know Ruby :P